Revit_Core_Engine/Query/StringValue.cs

```diff
@@ -31,14 +32,17 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
+        [Description("Extracts a string value from a given Revit Parameter.")]
+        [Input("parameter", "Revit Parameter to extract the string value from.")]
+        [Output("value", "String value extracted from the input Revit Parameter.")]
         public static string StringValue(this Parameter parameter)
         {
             if (parameter != null && parameter.HasValue)
             {
                 switch (parameter.StorageType)
                 {
                     case (StorageType.Double):
-                        return UnitUtils.ConvertFromInternalUnits(parameter.AsDouble(), parameter.DisplayUnitType).ToString();
+                        return UnitUtils.ConvertFromInternalUnits(parameter.AsDouble(), parameter.GetUnitTypeId()).ToString();
```

Comment:
I think I'm missing something. How does this compile when you set the Solution on any version > Revit2020, if it relies on `GetUnitTypeId()` that becomes unavailable?

Reply:
The method does not become unavailable - actually, it had been added to the API starting in 2021, so the one implemented in this PR is mimicking the new mechanism. I have just added [this link](https://thebuildingcoder.typepad.com/blog/2020/04/whats-new-in-the-revit-2021-api.html) to the PR description, please feel free to have a look at section 1.3 for the list of changes that I am trying to align to.

Reply:
For the record, as from our chat, I thought we should be more explicit on that by changing the description on the GetUnitTypeId() method.